### PR TITLE
docs: refactor README file

### PR DESCRIPTION
**Pull request by mattdibi**
This PR refactors the main README file to take into account changes added by https://github.com/eclipse/kura/pull/5060

### CODE REVIEW

**Comment by mattdibi** [comment]
@GregoryIvo feel free to give some feedback here :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ You do this by adding the `-s` flag when you make the commit(s), e.g.
 
 You can find all the details in the [Contributing via Git](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git) document on the Eclipse web site.
 
+#### Development Model
+
+Development on Kura™ follows a [variant of the gitflow model](https://github.com/eclipse/kura/wiki/New-Kura-Git-Workflow).  Development is made on the [develop branch](/eclipse/kura/tree/develop). The master branch is not used anymore.
```
Comment (MMaiero) [inline CONTRIBUTING.md]: Eclipse Kura

File: README.md [hunk 2]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
 
 ### Building Eclipse Kura™ Containers
 
-The kura™ container build process currently only supports x86 containers. Following the instructions below will build two containers. One based on Alpine Linux ```kura-alpine-x86_64```, and another on Ubi8 ```kura-ubi8-x86_64```.
+The kura™ container build process currently only supports x86 containers. Following the instructions below will build two containers. One based on Alpine Linux `kura-alpine-x86_64`, and another on Ubi8 `kura-ubi8-x86_64`.
```
Comment (MMaiero) [inline README.md]: Eclipse Kura

File: README.md [hunk 3]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
...
 
-Build Kura™ as per [our instructions](https://github.com/eclipse/kura#build-kura). To build the containers you'll need to change the target of the "Build the target profiles" step like the following:
+Build Kura™ as per [our instructions](#build-kura). To build the containers you'll need to change the target of the "Build the target profiles" step like the following:
```
Comment (MMaiero) [inline README.md]: Eclipse Kura

File: README.md [hunk 4]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
...
-- Set the `JRE 1.8 location` value to the installed local jdk-8 VM
-- Leave all Bootstrap Tasks selected and press the Finish button
-- Accept all the licenses and wait for the installation to finish
-At first startup Eclipse IDE will checkout the code, perform a full build and configure a few Working Sets
-- When the tasks are completed. In the IDE open (double click) Target Platform > Target-Definition > Kura™ Target Platform Equinox 3.16.0, and press "Set as Target Platform" located at the top right of the window
-
-Now you are ready to develop on Eclipse Kura™.
-
-To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+After this command runs, images can be found in your preferred container engine image list.
```
Comment (MMaiero) [inline README.md]: After the command run?

File: README.md [hunk 5]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
...
-### Known Issues
-Currently, the emulator web ui is not properly working on Windows so, with your setup, you will be able to build and deploy you applications, but not be able to use the Eclipse IDE based Kura™ emulator.
+IDE Setups
+----------
 
-The full build of Kura™ is only supported for Linux and Mac Os based systems.
+We currently support two setups for Kura™ development:
 
-Currently the maven build on Windows requires to disable the tests and will fail when it tries to create the installers for the target platforms.
+- [**Kura™ Development Environment Setup**](https://eclipse.github.io/kura/latest/java-application-development/development-environment-setup/): This is the full setup allowing you to contribute to the core Kura™ project codebase. It will install all the IDE plugins and formatters to have a pleasant development experience and clone the Kura™ source code on your workstation.
```
Comment (MMaiero) [inline README.md]: Eclipse Kura

File: README.md [hunk 6]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
...
-To raise an issue, please report a bug on [GitHub issues](https://github.com/eclipse/kura/issues/new).
+After this command runs, images can be found in your preferred container engine image list.
 
-### Known Issues
-Currently, the emulator web ui is not properly working on Windows so, with your setup, you will be able to build and deploy you applications, but not be able to use the Eclipse IDE based Kura™ emulator.
+IDE Setups
+----------
 
-The full build of Kura™ is only supported for Linux and Mac Os based systems.
+We currently support two setups for Kura™ development:
```
Comment (MMaiero) [inline README.md]: Eclipse Kura

File: README.md [hunk 7]
```diff
@@ -163,57 +180,28 @@ and select the profiles you want to build.
...
 
-Currently the maven build on Windows requires to disable the tests and will fail when it tries to create the installers for the target platforms.
+- [**Kura™ Development Environment Setup**](https://eclipse.github.io/kura/latest/java-application-development/development-environment-setup/): This is the full setup allowing you to contribute to the core Kura™ project codebase. It will install all the IDE plugins and formatters to have a pleasant development experience and clone the Kura™ source code on your workstation.
+- [**Add-on Development Environment Setup**](https://eclipse.github.io/kura/latest/java-application-development/kura-workspace-setup/): This setup will allow you to develop applications or bundles running on Kura™. It will install only the APIs and the examples and is best suited for developing Kura™ add-ons.
```
Comment (MMaiero) [inline README.md]: Eclipse Kura

File: README.md (hunk 4 above)
Comment (mattdibi) [reply README.md]: Addressed in: 53fc4ac471ea81e481b9d015959ecec57240273b

File: README.md (hunk 3 above)
Comment (mattdibi) [reply README.md]: Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3

File: README.md (hunk 2 above)
Comment (mattdibi) [reply README.md]: Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3

File: README.md (hunk 5 above)
Comment (mattdibi) [reply README.md]: Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3

File: README.md (hunk 6 above)
Comment (mattdibi) [reply README.md]: Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3

File: README.md (hunk 7 above)
Comment (mattdibi) [reply README.md]: Addressed in: 3fafedaed27d31d091bd64e67c809a32d5e77ac3

**Review by MMaiero — APPROVED** [review]

**Pull request merged**